header.php

```diff
@@ -57,11 +57,11 @@
     <div id="wrapper">
 
         <div id="header">
-        	<? include("header_social.php"); ?>
```

Comment:
Why did you add this PHP? It is working on my side?

Reply:
It's not working on Windows

Reply:
Great, works on my side with Ubuntu